src/Symfony/Component/Security/Core/Authentication/Token/AbstractToken.php

```diff
@@ -275,6 +292,9 @@ final public function unserialize($serialized)
         $this->__unserialize(\is_array($serialized) ? $serialized : unserialize($serialized));
     }
 
+    /**
+     * @deprecated since Symfony 5.4
```

Comment:
no runtime deprecation?

Reply:
the method is private, the docblock is just a reminder for later removal